src/core/backbone_router/bbr_local.cpp

```diff
@@ -52,7 +52,7 @@ Local::Local(Instance &aInstance)
     , mState(OT_BACKBONE_ROUTER_STATE_DISABLED)
     , mMlrTimeout(Mle::kMlrTimeoutDefault)
     , mReregistrationDelay(Mle::kRegistrationDelayDefault)
-    , mSequenceNumber(Random::NonCrypto::GetUint8())
+    , mSequenceNumber(Random::NonCrypto::GetUint8() % 127)
```

Comment:
should we avoid 126 ?

Reply:
What I understand from the specification is that 126 is possible as a random value.

Reply:
Agree